src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj

```diff
@@ -39,7 +39,7 @@
     <None Include="..\lib\Microsoft.SqlServer.TransactSql.ScriptDom.dll" Pack="true" PackagePath="lib/$(TargetFramework)" />
     <None Include="..\README.md" Pack="true" PackagePath="" />
   </ItemGroup>
-  <Target Name="CopyNupkg" AfterTargets="Pack">
+  <Target Name="CopyNupkg" AfterTargets="Build">
```

Comment:
I assume this won't run if you do `dotnet pack` then, right?

Reply:
By default, dotnet pack builds the project first. (https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-pack)

Reply:
But after it builds the package doesn't exist yet...it needs to actually complete the pack target. So this doesn't seem like the correct ordering here - if there is a nupkg it will contain old stuff since the newly built stuff hasn't been built yet.

Reply:
We have <GeneratePackageOnBuild>true</GeneratePackageOnBuild>, I assumed we'll have the package ready.

Reply:
Hmm, did you verify that? Does it actually generate it as part of the build task or just kick off the pack task after the build task is done?

Reply:
IIRC if you run with verbose logging it lists out all the targets it ran at the end, which should help understand what exactly happens.

Reply:
Will add the verbose logging and see the order, thanks!

Reply:
> Hmm, did you verify that? Does it actually generate it as part of the build task or just kick off the pack task after the build task is done?

After checking the verbose logs, GeneratePackageOnBuild will trigger pack command after build. Now am not sure why sometimes that step is missing from the logs on the pipeline (on the failed runs). I tried searching online but didn't come across anything that's helpful. Am trying to force the pack command to run explicitly after build and see if we have a consistent copy package.